train.py

```diff
@@ -0,0 +1,181 @@
+import json
+import random
+import time
+import warnings
+
+import spacy
+from spacy.util import compounding, minibatch
+from spacy.gold import GoldParse
+from spacy.scorer import Scorer
+from tqdm import tqdm
+import os
+import logging
+import pandas as pd
+import copy
+
+names = pd.read_csv('top100names.csv')
+
```

Comment:
```suggestion
logging.basicConfig(level=logging.INFO)
logger = logging.getLogger(__name__)
```
^ need this for logs to show in console